src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -33,6 +38,25 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
         checkArgument(preppedWord.split("\\s+").length == 1, "Word parameter should be a single word");
 
         String preppedSentence = sentence;
+
+        return new SearchPair(preppedSentence, preppedWord);
+    }
+
+    /**
+     * Returns true if the {@code sentence} contains the {@code word}.
+     *   Ignores case, but a full word match is required.
+     *   <br>examples:<pre>
+     *       containsWordIgnoreCase("ABc def", "abc") == true
+     *       containsWordIgnoreCase("ABc def", "DEF") == true
+     *       containsWordIgnoreCase("ABc def", "AB") == false //not a full word match
+     *       </pre>
```

Comment:
Should there be empty line between description and param?

Reply:
Will change but this was previous code from AB3.